resources/schema.json

```diff
@@ -122,6 +122,12 @@
                         "type": "string"
                     }
                 },
+                "global-ignoreSourceCodeByRegex": {
```

Comment:
Why not call it just `ignoreSourceCodeByRegex` ?

Reply:
to be consistent with the `global-ignore`

Reply:
To keep consistency with `ignore`/`global-ignore`?